plenum/server/consensus/ordering_service.py

```diff
@@ -487,6 +488,9 @@ def _enqueue_commit(self, request: Commit, sender: str):
             self.commitsWaitingForPrepare[key] = deque()
         self.commitsWaitingForPrepare[key].append((request, sender))
 
+    def _validate_primaries(self, pre_prepare: PrePrepare):
```

Comment:
As primaries are part of Audit txn, and we validate audit ledger root hash, we may skip this check. If we want to add a check (to have more explicit "Primaries are invalid" suspicion code instead of just "Incorrect audit ledger"), then the validation needs to be done not here, but in `_validate_applied_pre_prepare` method.